core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BillingAccountChangeSynchronizer.scala

```diff
@@ -101,12 +98,6 @@ final case class BillingAccountChangeSynchronizer(dataSource: SlickDataSource,
     for {
       billingProject <- loadBillingProject
 
-      // Try out the new query using the STATUS column and compare results. If validation fails, don't
-      // fail the entire process.
-      _ <- validateStatusColumn.recover { case e: Exception =>
-        logger.warn(s"BillingAccountChangeStatus logic failed with ${e.getMessage}", e)
-      }
-
```

Comment:
we no longer need to validate the old query and new query return the same results, since we're getting rid of the old query